solutions/src/main/java/ru/sberbank/school/task06/CollectionUtils.java

```diff
@@ -0,0 +1,169 @@
+     *
+     * @param c1 первая коллекция
+     * @param c2 вторая коллекция
+     * @return true, если все элементы коллекции c2 содержатся в c1
+     */
+    public static <T> boolean containsAll(List <? extends T> c1, List <? super T> c2) {
+        return c1.containsAll(c2);
+    }
+
+    /**
+     * Проверка на содержение хотя бы одного элемента одной коллекции в другой.
+     *
+     * @param c1 первая коллекция
+     * @param c2 вторая коллекция
+     * @return true, если хотя бы 1 элемент коллекции c2 содержатся в c1
+     */
+    public static <T> boolean containsAny(List <T> c1, List <T> c2) {
+        for (T o : c2) {
+            if (c1.contains(o)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Создание коллекции, содержащей элементы из входной коллекции в диапазоне от min до max (не
+     * индексы, а значения). Элементы необходимо сравнивать, через интерфейс Comparable Пример:
+     * range(Arrays.asList(8, 1, 3, 5, 6, 4), 3, 6) вернет {3,4,5,6}
+     *
+     * @param list исходная коллекция
+     * @param min  минимальное значение
+     * @param max  максимальнео значение
+     * @return отфильрованный по минимальному и максимальному значению список
+     */
+    public static <T extends Object & Comparable<? super T>> List<T> range(List <? extends T> list, T min, T max) {
+        List<T> resultList = new ArrayList<>();
+
+        for (T elem : list) {
+            if (elem.compareTo(min) >= 0 && elem.compareTo(max) <= 0) {
+                if (!resultList.isEmpty()) {
+                    int index = resultList.size();
+                    if (elem.compareTo(resultList.get(index - 1)) <= 0) {
+                        for (T current : resultList) {
+                            if (elem.compareTo(current) <= 0) {
+                                index = resultList.indexOf(current);
+                                break;
+                            }
+                        }
+                        resultList.add(index, elem);
+                    } else {
+                        resultList.add(elem);
+                    }
+                } else {
+                    resultList.add(elem);
+                }
+            }
+        }
+        return resultList;
+    }
+
+    /**
+     * Создание коллекции, содержащей элементы из входной коллекции в диапазоне от min до max (не
+     * индексы, а значения). Элементы необходимо сравнивать, через интерфейс Comparator Пример:
+     * range(Arrays.asList(8, 1, 3, 5, 6, 4), 3, 6) вернет {3,4,5,6}
+     *
+     * @param list исходная коллекция
+     * @param min  минимальное значение
+     * @param max  максимальнео значение
+     * @return отфильрованный по минимальному и максимальному значению список
+     */
+    public static <T> List<T> range(List<? extends T> list, T min, T max, Comparator<? super T> comparator) {
+        List<T> resultList = new ArrayList<>();
+
+        for (T elem : list){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-632789073>*